rpc/send_test.go

```diff
@@ -156,6 +156,31 @@ func TestUnretryableError(t *testing.T) {
 	}
 }
 
+// TestClientNotReady verifies that Send gets an RPC error when a client
+// does not become ready.
+func TestClientNotReady(t *testing.T) {
+	rpcContext := createNewTestRPCContext(t)
+
+	opts := Options{
+		N:               1,
+		Ordering:        OrderStable,
+		SendNextTimeout: 1 * time.Second,
+		Timeout:         1 * time.Second,
```

Comment:
this test looks like it's going to run for a second. We definitely don't want that. Set the timeout to a Nanosecond so that the test pretty much immediately fails.